aalto/jupyterhub-data.rst

```diff
@@ -3,34 +3,63 @@ Accessing JupyterHub data
 =========================
 
 Unlike many JupyterHub deployments, your data is *yours* and have many
-different ways to access it.  Most importantly, the data is a normal
-Aalto network drive, and thus it can be accessed remotely, from your
-own computers.
+different ways to access it.  Thus, we don't just have jupyter.cs, but
+a whole constellation of ways to access and do *your work*, depending on what
+suits you best for each part.
+
+Your data (and as an instructor, your course's data) can be accessed
+many ways:
+
+* On jupyter.cs.
+* Via network drive *on your own computer as local files*.
+* On Aalto shell servers (such as kosh.aalto.fi).
+* On other department/university workstations.
 
 On Paniikki and Aalto computers
 -------------------------------
 
-TODO: eventually, on Paniikki, and the Aalto servers kosh, lyta,
-brute, and force, the JupyterHub data will be available
-automatically.
+On Paniikki, and the Aalto servers ``kosh.aalto.fi``,
+``lyta.aalto.fi``, ``brute.aalto.fi``, and ``force.aalto.fi`` (and
+possibly more), the JupyterHub is available automatically.  **You can,
+for example, use the Paniikki GPUs.**
+
+Data is available within the paths ``/m/jhnas/jupyter``.
+
+.. csv-table::
+   :delim: |
+   :header-rows: 1
+
+   Name                    | Path on Linux servers
+   personal notebooks      | ``/m/jhnas/jupyter/u/$nn/$username/``
+   course data             | ``/m/jhnas/jupyter/course/$course_slug/data/``
+   course instructor files | ``/m/jhnas/jupyter/course/$course_slug/files/``
+   shared data             | ``/m/jhnas/jupyter/shareddata/``
 
```

Comment:
Maybe a summary of these attributes could be nice, but maybe not necessary. Other than that very good summary.

Reply:
What kind of summary?  Do you mean what does `personal notebooks`, `course data`, etc means in the user servers?  If so, good idea... we could include the paths.

Reply:
Yeap, something like that would probably be sufficient.

Reply:
Maybe also a quick summary of `$course_slug`, `$nn` and `$username` would not hurt either.

Reply:
Updated, both things added.  I added a new table for `$nn` and the like - do you think that's too much space devoted to it?

Reply:
Nope. This is fine.